packages/design-system/src/components/Form/Primitives/Field/Field.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { forwardRef, ReactElement, Ref, cloneElement } from 'react';
+import Link, { LinkProps } from '../../../Link/Link';
+import { StackVertical } from '../../../Stack';
+import Label, { LabelProps } from '../Label/Label';
+import { InlineMessageDestructive, InlineMessageInformation } from '../../../InlineMessage';
+import VisuallyHidden from '../../../VisuallyHidden';
+
+export type FieldStatusProps = {
+	isError?: boolean;
```

Comment:
I'm not too keen on that wording, sounds wrong to me (not the first time I see it so I might be wrong)
```suggestion
	hasError?: boolean;
```

Reply:
It's just a standard I'm used to follow for boolean. All booleans regardless of what they are start with `is`. I wish there was a Talend coding styleguide to settle on that.

Reply:
The best is linter for this. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md

Reply:
That linter allows for both. If you guys are ok with both, fine by me!